src/ivoc/ocpicker.cpp

```diff
@@ -103,16 +104,14 @@ void StandardPicker::event(const Event& e) {
 }
 
 void StandardPicker::unbind(int m, EventButton eb) {
-    long cnt = handlers_[m]->size();
-    for (long i = 0, j = 0; i < cnt; ++i) {
-        ButtonHandler* b = handlers_[m]->at(j);
+    for (auto& b: *handlers_[m]) {
```

Comment:
```suggestion
    for (ButtonHandler * const b: *handlers_[m]) {
```
I know it's not exactly modern; but it might be very useful to have the type information visible.